community/concepts/bikeshedding.md

```diff
@@ -0,0 +1,22 @@
+### bikeshedding
```

Comment:
Should be capitalized? The other glossary terms are...

Reply:
It's not consistent, https://github.com/sourcecred/docs/pull/22/files#diff-4891a789c73197e20439da7feb2013ffR1 is not. I'm indifferent one way or the other, I think my original preference was lowercase because they are just terms, not formal titles or pronouns.

Reply:
Done.